openmc/capi/cell.py

```diff
@@ -128,6 +132,19 @@ def fill(self, fill):
             indices = (c_int32*1)(-1)
             _dll.openmc_cell_set_fill(self._index, 1, 1, indices)
 
+    def get_temperature(self, instance=None):
+        """Get the temperature of a cell
+
+        Parameters
+        ----------
+        instance: int or None
+            Which instance of the cell
+
+        """
+        T = c_double()
+        _dll.openmc_cell_get_temperature(self._index, c_int32(instance), T)
```

Comment:
`c_int32(instance)` isn't happy taking `None`. We'll need a check here for that:

```python
if instance is not None:
    instance = c_int32(instance)
_dll.openmc_cell_get_temperature(self._index, instance, T)
```

Reply:
Apologies for the delay on my end, I'm new to this capi module. Would there not also be a problem with `instance` taking `None` in `set_temperature`? It's not clear to me why `get_temperature` would need this check but `set_temperature` wouldn't

Reply:
Very fair question. `set_temperature` *does* need this check. It currently won't work without a value for `instance`. After confirming, would you mind adding a similar check there as well?

Reply:
On that topic, you should add a test somewhere in `tests/unit_tests/test_capi.py` for get_temperature, and perhaps we ought to add the `None` cases there too.